x/marker/spec/01_state.md

```diff
@@ -143,6 +148,14 @@ transfer the marker's coins from another account unless granted permission to do
 
 Markers with **Coin** type cannot be configured to allow forced transfers.
 
+### Required Attributes
+
+A marker with the **Restricted Coin** type can be configured to allow transfers with a normal `MsgSend` to address that have defined attributes. 
+This can be configured by setting the `restricted_attributes` array on the Marker.  When a `MsgSend` transaction is executed and the coin type is `restricted`, the `required_attributes` are checked. If the `ToAddress` associated with the `MsgSend` command has **all** the required attributes, the transfer will be executed.
```

Comment:
Should `restricted_attributes` be `required_attributes`?

Reply:
Yes, I kept making this mistake in code too.  Thanks for catching that!